### PR TITLE
update browser test to use dusk for #249

### PR DESCRIPTION
**Pull request by EvanDime**
##### **_IMPORTANT: Always create an issue before a PR and provide enough information so that others can review_**

update feature's browser test to work with dusk
##### _What existing problem does the pull request solve?_
Improved browser test

##### _Demonstrate the code is solid. </br> Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI._

This is the test
##### _List all issues this PR resolves_
- applies to #287 
##### _Complete these items before requesting reviews_
- [x] Static Analysis Rules Validated
- [x] All commits follow naming conventions

### Sub-tasks Checklist
- [x] UI Implementation
- [x] UI Test
- [x] Backend Implementation
- [x] Backend Tests
- [x] Integration Tests
- [x] System Tests
- [x] Documentation

### CODE REVIEW

**Comment by EvanDime** [comment]
tests are failing since these tests are waiting on #544 to be merged (was working on local since i never rebuilt the FE assets)

File: database/migrations/2021_01_24_170653_create_reservations_table.php
Comment (alexstojda) [inline database/migrations/2021_01_24_170653_create_reservations_table.php:47]: errr, this smells like a very bad practice. Why is truncation needed?

File: tests/Browser/Pages/Bookings.php
Comment (alexstojda) [inline tests/Browser/Pages/Bookings.php:42]: This class can be removed, a `Page` is only useful if you are adding some re-used functionality (like filling a form for example, or clicking certain things)

**Review by alexstojda — CHANGES_REQUESTED** [review]
smol tings

File: database/migrations/2021_01_24_170653_create_reservations_table.php
Comment (EvanDime) [reply database/migrations/2021_01_24_170653_create_reservations_table.php:47]: because while we undo reservations we can't convert them back to bookings since restrictions allow bookings to have multiple times. When rolling back, it can only have one time at a time. Also we are dropping the reservations table already. because of this we need to truncate the bookings requests so that the foreign keys aren't null. All live booking requests that exist should be deleted when you delete all the times associated with them

Comment (alexstojda) [reply database/migrations/2021_01_24_170653_create_reservations_table.php:47]: Explanation checks out 👍

**Review by alecadub — APPROVED** [review]
looks good after @alexstojda comments !

File: tests/Browser/Pages/Bookings.php
Comment (EvanDime) [reply tests/Browser/Pages/Bookings.php]: Removed the page. Once the tests pass feel free to approve.

Comment (EvanDime) [reply tests/Browser/Pages/Bookings.php]: @alexstojda

**Review by alexstojda — APPROVED** [review]

**Pull request merged**
